evergreen/config.yml

```diff
@@ -1761,12 +1776,25 @@ buildvariants:
   - name: compile_test_coverage
   - name: finalize_coverage_data
 
+- name: ubuntu-no-app-services
+  display_name: "Ubuntu (AppServices Disabled)"
+  run_on: ubuntu2204-arm64-large
+  expansions:
+    cmake_url: "https://s3.amazonaws.com/static.realm.io/evergreen-assets/cmake-3.26.3-linux-aarch64.tar.gz"
+    cmake_bindir: "./cmake_binaries/bin"
+    fetch_missing_dependencies: On
+    c_compiler: "/opt/clang+llvm/bin/clang"
+    cxx_compiler: "/opt/clang+llvm/bin/clang++"
+    extra_flags: "-DREALM_APP_SERVICES=OFF"
+    disable_tests_against_baas: On
+  tasks:
+  - name: compile_local_tests
+
 - name: windows-64-vs2019
   display_name: "Windows x86_64 (VS 2019)"
   run_on: windows-vsCurrent-large
   expansions:
-    cmake_url: "https://s3.amazonaws.com/static.realm.io/evergreen-assets/cmake-3.26.3-windows-x86_64.zip"
-    cmake_bindir: "./cmake-3.26.3-windows-x86_64/bin"
+    cmake_bindir: "/cygdrive/c/Program Files/CMake/bin"
```

Comment:
I believe we used to rely on a downloaded version of CMake because the locally installed version on these machines was below our requirement (currently 3.22.1) but it appears that all runs were successful so the versions at these paths is correct. I checked one run log and it was using 3.28.3. Using the built-in version is much easier than maintaining a binary download ourselves so this is a nice improvement 👍  That and it is also more efficient for the CI machines.